site/content/configuration/health-checks.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Health Checks"
```

Comment:
```suggestion
title: "Health Checks"
```

We use Sentence case in titles and headers.

I will open a separate PR to correct all the docs titles once this is merged.

Reply:
OK. Thanks @Jcahilltorre